src/gl/gl_builders.js

```diff
@@ -134,9 +134,59 @@ GLBuilders.buildExtrudedPolygons = function (
     }
 };
 
+// Add to equidistant pairs of vertices
+function addVertexPair (coord, normal, v_pct, { halfWidth, vertices, scalingVecs, texcoords, min_u, min_v, max_u, max_v }) {
```

Comment:
Maybe add a comment for these functions indicating that they are internal helper methods for the polyline builder. Maybe put them after the main polyline builder method.